plugins/CoreAdminHome/Tasks.php

```diff
@@ -52,6 +59,57 @@ public function schedule()
         if(SettingsPiwik::isInternetEnabled() === true){
             $this->weekly('updateSpammerBlacklist');
         }
+
+        $this->scheduleTrackingCodeReminderChecks();
+    }
+
+    private function scheduleTrackingCodeReminderChecks()
+    {
+        // add check for a site's tracked visits
+        $sites = Request::processRequest('SitesManager.getAllSites');
+
+        $daysToTrackedVisitsCheck = (int) Config::getInstance()->General['num_days_before_tracking_code_reminder'];
+        if ($daysToTrackedVisitsCheck <= 0) {
+            return;
+        }
+
+        foreach ($sites as $site) {
+            $createdTime = Date::factory($site['ts_created']);
+
+            $scheduledJobTime = $createdTime->addDay($daysToTrackedVisitsCheck);
+            if ($scheduledJobTime->isEarlier(Date::now())) {
```

Comment:
I've been looking at this code for quite a long time and trying to understand it... I know I asked you before and you have tests but are you sure this works? I set the time to email to 1 day in config, had a created site from `Sep 14th`, today is `Sep17th` and it wouldn't email me. Only after I changed to `isLater` it sent me that email notification. Not sure I'm seeing it correct... and I still don't get the monthly below even after your explanation how it will be sent only once :)

![image](https://user-images.githubusercontent.com/273120/45602611-584a3b80-ba75-11e8-8d83-969fd5329033.png)

I might just not see it though.

Reply:
Will try to clarify.